.gitignore

```diff
@@ -0,0 +1 @@
+.env
```

Comment:
add node modules to git ignore (node modules should not be pushed to master later)